js/ui/drop_down_editor/ui.drop_down_list.d.ts

```diff
@@ -107,7 +107,7 @@ export interface dxDropDownListOptions<TComponent> extends DataExpressionMixinOp
      * @action
      * @public
      */
-    onValueChanged?: ((e: NativeEventInfo<TComponent> & ValueChangedInfo) => void);
+    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | UIEvent> & ValueChangedInfo) => void);
```

Comment:
```suggestion
    onValueChanged?: ((e: NativeEventInfo<TComponent, KeyboardEvent | MouseEvent | PointerEvent | Event> & ValueChangedInfo) => void);
```

https://github.com/DevExpress/DevExtreme/pull/19950/files#r741000230